client/question/components/SpeechBalloonsContainerDirective.js

```diff
@@ -22,7 +22,7 @@ tie.directive('speechBalloonsContainer', [function() {
     scope: {},
     template: `
       <div>
-        <div class="tie-dot-container" ng-class="{'tie-display-dots': ConversationLogDataService.isNewBalloonPending()}"}>
+        <div class="tie-dot-container" ng-class="{'tie-display-dots': (ConversationLogDataService.data.speechBalloonList.length > 0 ? ConversationLogDataService.data.speechBalloonList[0].isDisplayedOnRight() : false)}"}>
```

Comment:
Might be better to avoid the ternary and just use ConversationLogDataService.data.speechBalloonList.length > 0 && ConversationLogDataService.data.speechBalloonList[0].isDisplayedOnRight()